Automattic/wp-calypso
# Jetpack: Remove canManage and remoteManagementUrl

Currently, we compute the `canManage` prop for every Jetpack site. This prop was necessary back in time to signify that Jetpack Manage is available to enable remote management. This prop was available since Jetpack 3.4. However, since Jetpack 7.3 (released almost a year ago), Manage no longer exists and is considered always on (and not disable-able), making this prop unnecessary.

Jetpack's official support policy is to support the current - 1 version. Any older versions aren't supported. This means we can remove any checks that check for 7.9-only or older versions of Jetpack. This basically means removing all of those `canManage` usages. This PR does that, and also removes all of the related components, selectors and tests that are no longer used. 

This PR is a follow-up to #38585, where we did similar cleanup for no longer supported versions of Jetpack.

#### Changes proposed in this Pull Request

* Remove any `canManage` usage from Themes, Plugins and Jetpack Connect
* Delete obsolete `JetpackManageDisabledMessage`
* Delete obsolete `PluginSiteDisabledManage`
* Delete obsolete `canJetpackSiteManage` and `getJetpackSiteRemoteManagementUrl` selectors.
* Update (and clean up) some tests.

#### Testing instructions

* Do some smoke testing with the affected sections for Jetpack sites and look out for errors:
  * Plugins
  * Themes
  * Jetpack Connect
* Verify tests pass
